roulette/src/components/movieError/ErrorBoundary.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import MovieError from './MovieError';
+
+/**
+ * Error Boundary to route to error component when error occurs in the child component.
+ */
+class ErrorBoundary extends React.Component {
+  state = { error: null };
+
+  /**
+   * Adds two numbers together.
+   * @param {object} error The error object.
+   * @return {object} The error object.
+   */
+  static getDerivedStateFromError(error) {
+    return { error };
+  }
+
+  /**
+   * Adds two numbers together.
+   * @return {any} The error component or the continue to the right component.
+   */
+  render() {
+    const { error } = this.state;
+    if (error) {
+      return <MovieError error={error} />;
```

Comment:
Since you already have a working Error Boundary solution, consider also addressing the error that the project has at the moment.
<img width="854" alt="Screenshot 2022-05-03 at 14 18 04" src="https://user-images.githubusercontent.com/25229823/166460189-bdf19cc3-4397-455c-aabd-5ad92be9fb91.png">

This is after running:
`cd netflix-react/roulette`
`npm i`
`npm start`

My Node.js version is 16.14.1

Reply:
guess this has something to do with the reach ui library I used, but I was able to remove the error by 
setting strictExportPresence: false in the webpack config in the node_module which I cant push to the repo.